FWCore/ParameterSet/python/Config.py

```diff
@@ -513,6 +574,8 @@ def addSubProcess(self,mod):
         self.__subProcesses.append(mod)
     def _placeService(self,typeName,mod):
         self._place(typeName, mod, self.__services)
+        if typeName in self.__dict__:
+          self.__dict__[typeName]._inProcess = False
```

Comment:
PathValidator and a few other similar classes check that the elements in a Path are attributes of the process. They usually do this by checking that the module has a label. Now one can put a Service on a Task, but services do not have labels. So to perform the same check I added a new variable that indicates whether a service is attached to the process. Its name is '_inProcess'. The particular line above is setting this false when a service that was attached to the process is being replaced by a different instance of that same service.